arrows/mvg/applets/CMakeLists.txt

```diff
@@ -24,6 +26,7 @@ kwiver_add_plugin( kwiver_algo_mvg_applets
   SOURCES      ${sources} ${headers}
   PRIVATE      kwiver_algo_core
                kwiver_algo_mvg
+               kwiver_algo_vtk  
```

Comment:
@danlipsa the intent was for this tool to go in `vtk/applets` instead of `mvg/applets` so that mvg does not need to depend on VTK.  I know `vtk/applets` does not exist yet, but we need to create it using the same pattern as `mvg/applets`.

Reply:
OK, sounds good. I'll move it there.